ocpmodels/trainers/forces_trainer.py

```diff
@@ -601,6 +601,10 @@ def _compute_metrics(self, out, batch_list, evaluator, metrics={}):
     def run_relaxations(self, split="val"):
         ensure_fitted(self._unwrapped_model)
 
+        # When set to trie, uses deterministic CUDA scatter ops, if available.
+        # https://pytorch.org/docs/stable/generated/torch.use_deterministic_algorithms.html#torch.use_deterministic_algorithms
+        registry.register("set_deterministic_scatter", False)
```

Comment:
Should we make this easily accessible from the config? i.e.

`registry.register("set_deterministic_scatter", self.config.get("set_deterministic_scatter", False))`